src/app/routes/api/v3/forms/__tests__/public-forms.verification.routes.spec.ts

```diff
@@ -297,6 +386,113 @@ describe('public-forms.verification.routes', () => {
       expect(response.text).toBe(getReasonPhrase(StatusCodes.CREATED))
     })
 
+    it('should return 201 when using SingPass auth and user is logged in to SingPass', async () => {
+      // Arrange
+      const mockJwt = 'mockJwt'
+
+      jest.spyOn(SpOidcClient.prototype, 'verifyJwt').mockResolvedValueOnce({
+        userName: 'S1234567A',
+      })
+
+      // Act
+      const response = await request
+        .post(
+          `/forms/${mockSPFormId}/fieldverifications/${mockTransactionId}/fields/${mockEmailFieldId}/otp/generate`,
+        )
+        .send({
+          answer: 'mail@me.com',
+        })
+        // Note cookie is for SingPass, not CorpPass
+        .set('Cookie', [`jwtSp=${mockJwt}`])
```

Comment:
nit: should we use the enum to reduce this test's brittleness?

```suggestion
        .set('Cookie', [`${JwtName.SP}=${mockJwt}`])
```

also for the rest hehe